_posts/2021-09-20-newpipe-0.21.10-released.md

```diff
@@ -0,0 +1,112 @@
+
+- [@litetex](https://github.com/litetex), being on a roll, reworked the error panel shown when an error happens. This results in a clearer error display.
+
+- [@Redirion](https://github.com/Redirion) implemented proper closure of audio streams when playback is stopped. This fixed a longstanding bug where NewPipe would create more and more audio sessions but use only one. This messed with equaliser apps.
+
+- [@TobiGr](https://github.com/tobigr) added proper error handling which might occur when trying to play a video in popup mode or get a search suggestion.
+
+- [@TobiGr](https://github.com/tobigr) also took care of a small UI bug: when one played something in the miniplayer and changed the visibility of the description tab in the meantime, the visibility of the comments tab was changed instead. The app isn't confused anymore, we cannot say the same about the dev...
+
+## Translation
+
+- [@Stypox](https://github.com/Stypox) switched to using the custom cancel string everywhere, which is translated to more languages than the default android cancel string.
+
+- They jumped onto that plane and replaced a couple more strings to better translated ones: "ok", "yes", "no", and "cancel". They also removed "add", and renamed "finish" to "ok", because it clearly means "ok", not "finish"!
+
+- [@Aga-C](https://github.com/Aga-C) added plural forms for download-related strings.
+
+- [@CBSkarmory](https://github.com/CBSkarmory) improved some wording in the issue template, thanks!
+
+## Nerd Talk
+
+- [@TobiGr](https://github.com/TobiGr) updated the hardcoded SoundCloud client ID.
+
+- [@FireMasterK](https://github.com/FireMasterK) added support for extracting the channel avatar on YouTube and PeerTube.
+
+- [@FireMasterK](https://github.com/FireMasterK) also added the `uploaderVerified` info to the `StreamInfo` class.
+
+- [@litetex](https://github.com/litetex) switched GitHub Actions to using the Eclipse Temurin Java distribution, as AdoptOpenJDK is getting deprecated.
+
+- [@Redirion](https://github.com/Redirion) replaced NewPipe’s custom buffer values with ExoPlayer's default ones.
+
+- [@Isira-Seneviratne](https://github.com/Isira-Seneviratne) changed `PackageInfo` to its backwards-compatible version, `PackageInfoCompat` to fetch the app signatures.
+
+- [@litetex](https://github.com/litetex) also updated the version of Kotlin used everywhere, and fixed some build problems.
+
+- They also introduced a cache setting in the CI configuration which lets it run faster.
+
+- [@XiangRongLin](https://github.com/XiangRongLin) added a gradle parameter to prevent KtLint, the Kotlin [linter](https://en.wikipedia.org/wiki/Lint_(software))+[formatter](https://en.wikipedia.org/wiki/Prettyprint#Programming_code_formatting), from being run twice in the GitHub Actions [CI](https://en.wikipedia.org/wiki/Continuous_integration).
```

Comment:
```suggestion
- [@XiangRongLin](https://github.com/XiangRongLin) added a Gradle parameter to prevent KtLint, the Kotlin [linter](https://en.wikipedia.org/wiki/Lint_(software))+[formatter](https://en.wikipedia.org/wiki/Prettyprint#Programming_code_formatting), from being run twice in the GitHub Actions [CI](https://en.wikipedia.org/wiki/Continuous_integration).
```

Also, CI is mentioned in the previous entry, so I think that is where the Wikipedia article on CI should be linked.